.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+**PLEASE REMOVE THIS TEMPLATE BEFORE SUBMITTING**
+
+Before submitting a PR, please, make sure that:
+
+- Changes are in a separate branch, not in master.
+- The branch contains only one commit on top of master (if not, squash them into
+  one commit).
+- The commit has a descriptive commit message with a concise title (first line).
+- The commit message contains `fixes #XXXX` or `closes #XXXX` to auto-close the
+  issue(s) that the PR fixes (if any).
```

Comment:
We could also mention the sign-off line here for commit messages, I know it is in the Contributing.md but could be helpful here. What do you thing?

Reply:
Well, it's not only in CONTRIBUTING.md but it's also checked by run-tests.py . So, if a contributor really goes through this checklist, a missing signoff will be caught.